src/assets/scripts/client/InputController.js

```diff
@@ -122,6 +122,7 @@ export default class InputController {
         this.input.mouseDown = [0, 0];
         this.input.isMouseDown = false;
 
+        // TODO: Do we want setupHandlers to be `enableHandlers`, and called from `.enable()`?
```

Comment:
no. the `setupHandlers` method is not needed here.  I added these in error (all over the place 😱).

adding/removing event listeners should happen within `.enable()` and `disable()` and the two handler methods can be removed.

In es5 land, the `.setupHandlers()` method was used to properly bind event handlers: 
```javascript
this._onClickHandler = this.$element.on('click', this._onClick).bind(this);
```

With arrow functions, we don't need to do this anymore so these methods are no longer needed.

Reply:
Ah, it all makes sense now. 👍 

Removed the `setupHandlers` and `disableHandlers` methods and moved the stuff to `.enable()` and `.disable()`.